### ISSUE TITLE
1-org Github steps missing

### ISSUE THREAD

**Issue by lpezet** [terraform-google-modules/terraform-example-foundation#1303]
### TL;DR

Going through entire process and managed to get 0-bootstrap succeed without any issues.
When creating PR from `plan` into `production` branch, google-github-actions/auth action fails with error:
```
Error: google-github-actions/auth failed with: retry function failed after 4 attempts: the GitHub Action workflow must specify exactly one of "workload_identity_provider" or "credentials_json"! If you are specifying input values via GitHub secrets, ensure the secret is being injected into the environment. By default, secrets are not passed to workflows triggered from forks, including Dependabot.
```

### Expected behavior

Github Action google-github-actions/auth succeeds

### Terraform Configuration

```hcl
NA
```

### Terraform Version

```sh
NA
```

### Additional information

Checked documentation from 1-org directly:
https://github.com/terraform-google-modules/terraform-example-foundation/blob/master/1-org/README.md#deploying-with-github-actions
Checked 0-bootstrap Github documentation as well:
https://github.com/terraform-google-modules/terraform-example-foundation/blob/master/1-org/README.md#deploying-with-github-actions

None are providing steps to setup Github repo for 1-org (as it was setup for 0-bootstrap).
